buddy_alloc.h

```diff
@@ -948,10 +1049,15 @@ static unsigned char *buddy_main(struct buddy *buddy) {
 }
 
 static unsigned int buddy_relative_mode(struct buddy *buddy) {
-    return buddy->buddy_flags & BUDDY_RELATIVE_MODE;
+    return (unsigned int)buddy->buddy_flags & BUDDY_RELATIVE_MODE;
```

Comment:
buddy_flags is size_t.

Reply:
size_t is a platform-dependent unsigned int and since BUDDY_RELATIVE_MODE highest bit is 1 this conversion does not affect the code behavior in any way.